static/bidder-info/relevantdigital.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://{{.Host}}.relevant-digital.com/openrtb2/auction"
```

Comment:
This is not recommended. We highly discourage folks to use dynamic domain this way. Please review - https://github.com/prebid/prebid-server/issues/2606

Reply:
Hi @gargcreation1992 and thanks for reviewing!

For consistency between our adapters, I hope we could allow dynamic subdomains just as discussed here: https://github.com/prebid/Prebid.js/pull/9685#discussion_r1159833733

The main problem is that existing customers have prebid server clusters running at certain subdomains. Adding the proposed method in #2606 of query parameters and load balancer endpoint would add an extra request and that will have negative impact on the customers latency.